tests/e2e/volumepropverification_test.go

```diff
@@ -123,7 +123,7 @@ func TestVolumeProperties(t *testing.T) {
 				}
 				log.Println("Finished verifying volume properties like size, disk-format and attached-to-vm fields"+
 					" at vm and esx for volume - ", volName)
-				verification.ExecCmd(vms[vmIndx], "docker stop "+containerName+" ; docker rm "+containerName)
+				ssh.InvokeCommand(vms[vmIndx], "docker stop "+containerName+" ; docker rm "+containerName)
```

Comment:
Also please use "dockerHosts" in place of vms. In other tests dockerHosts is being used as the array of endpoints to use. Or standardize on a name - endpoint, vms, dockerHost (preferrable as it has a meaning in these tests).